dotnet/samples/Concepts/Agents/ChatCompletion_FunctionTermination.cs

```diff
@@ -23,7 +23,7 @@ public async Task UseAutoFunctionInvocationFilterWithAgentInvocationAsync()
             {
                 Instructions = "Answer questions about the menu.",
                 Kernel = CreateKernelWithFilter(),
-                ExecutionSettings = new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions },
+                Arguments = new(new OpenAIPromptExecutionSettings() { ToolCallBehavior = ToolCallBehavior.AutoInvokeKernelFunctions }),
```

Comment:
nit: I love the new() syntax when the left-hand side makes the type clear, but in this case there is no way to determine the type without jumping to the definition of the ChatCompletionAgent constructor.

Reply:
Yes, readability is important.  Adding type.